test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestProjectContext.cs

```diff
@@ -489,8 +476,24 @@ public XDocument GetXML()
 
                 foreach (var frameworkInfo in Frameworks)
                 {
+                    // Add TFM properties
+                    var tfmProps = new Dictionary<string, string>(frameworkInfo.Properties);
+
+                    if (!(Type == ProjectStyle.PackageReference && ToolingVersion15))
+                    {
+                        tfmProps.Add("TargetFrameworkIdentifier", frameworkInfo.Framework.Framework);
+                        tfmProps.Add("TargetFrameworkVersion", $"v{NormalizeVersionString(frameworkInfo.Framework.Version)}");
+                        tfmProps.Add("_TargetFrameworkVersionWithoutV", NormalizeVersionString(frameworkInfo.Framework.Version));
```

Comment:
what's this for?

Reply:
Our test infra pretends like we have an sdk based project...because we are not yet testing against one that can infer props...so I needed to do this to make the tests pass on 3.1.xxx. 

On another note...we should stop 3.1.xxx tests